pyproject.toml

```diff
@@ -16,7 +16,7 @@ databooks = "databooks.cli:app"
 python = "^3.7"
 typer = ">=0.4.0,<1.0.0"
 rich = "^12.6.0"
-pydantic = "^1.8.2"
+pydantic = "2.1.*"
```

Comment:
Can we make this a bit less restrictive?
i.e.: `>=2.1`?

Reply:
We could, but the issue is that it then might install 2.2, 2.3 etc... As I said:

For now, its kept at the version 2.1 to ensure the PR is small-ish, since >= 2.2 requires even a bit more changes (mostly since rootmodels cannot have extra fields anymore)...

wdyt?

Reply:
Ah yeah I see what you're saying
It's fine then but we should create another issue to try to use another Pydantic version to relax a bit the constraints
Even though I'm not sure anyone uses it as a dependency in their projects